srcpkgs/opensm/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'opensm'
+pkgname=opensm
+version=3.3.21
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf automake bison flex libtool rdma-core"
+short_desc="InfiniBand Subnet Manager and Administrator"
+maintainer="Rich G <rich@richgannon.net>"
+license="GPL-2.0, BSD"
```

Comment:
Please apply the proposed changes:

- Use the correct SPDX short-identifier for GPL-2.0 (it is missing -or-later or -only identifier)
- Use the correct SPDX short-identifier for 'BSD' (You need to check to see what BSD that is)

Reply:
Corrected in new commit.